python/lsst/daf/butler/core/ddl.py

```diff
@@ -31,21 +31,39 @@
 """
 from __future__ import annotations
 
-__all__ = ("TableSpec", "FieldSpec", "ForeignKeySpec", "Base64Bytes", "Base64Region")
+__all__ = ("TableSpec", "FieldSpec", "ForeignKeySpec", "Base64Bytes", "Base64Region",
+           "AstropyTimeNsecTai")
 
 from base64 import b64encode, b64decode
+import logging
 from math import ceil
 from dataclasses import dataclass
 from typing import Optional, Tuple, Sequence, Set
 
 import sqlalchemy
+import astropy.time
 
 from lsst.sphgeom import ConvexPolygon
 from .config import Config
 from .exceptions import ValidationError
 from .utils import iterable, stripIfNotNone, NamedValueSet
 
 
+_LOG = logging.getLogger(__name__)
+
+# These constants can be used by client code
+EPOCH = astropy.time.Time("1970-01-01 00:00:00", format="iso", scale="tai")
+"""Epoch for calculating time delta, this is the minimum time that can be
+stored in the database.
+"""
+
+MAX_TIME = astropy.time.Time("2038-01-19 03:14:07", format="iso", scale="tai")
```

Comment:
I'm fine with this being 2100 or even higher. We are using nanosecond integers so 32bit seconds doesn't mean anything.